charts/jaeger-v2/Chart.lock

```diff
@@ -0,0 +1,15 @@
+dependencies:
+- name: cassandra
+  repository: https://charts.helm.sh/incubator
```

Comment:
consider if we can also use bitnami for Cassandra -- https://github.com/bitnami/charts/tree/main/bitnami/cassandra

Reply:
do we have support for v5 of Cassandra
``` 
$helm search repo bitnami/cassandra
NAME                    CHART VERSION   APP VERSION     DESCRIPTION                                       
bitnami/cassandra       12.0.3          5.0.1           Apache Cassandra is an open source distributed ...
```

Reply:
yes